packages/blockchain-wallet-v4-frontend/src/components/Brokerage/ConfirmAvailabilityRows/util.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { add, format } from 'date-fns'
+
+import { DepositTerms, DisplayMode } from 'data/types'
+
+const Switcher = (
```

Comment:
This can be split into two subcomponents (e.g. `Title` and `Value`) to avoid variable reassignment.

Reply:
To simplify this further, `Switcher` could be a hook just returning the text. Do we have a hook solution for internationalization?